monkey/monkey_island/cc/ui/src/components/IslandHttpClient.tsx

```diff
@@ -54,6 +54,21 @@ class IslandHttpClient extends AuthComponent {
       })
       .then(res => new Response(res, status));
   }
+
+  getBytes(endpoint: APIEndpoint, args: Record<string, any>={}, refreshToken: boolean = false): Promise<Response> {
+    let status = null;
+    let params = new URLSearchParams(args);
+    let url = String(endpoint);
+    if(params.toString() !== ''){
+      url = endpoint + '?' + params;
+    }
+    return this.authFetch(url, {}, refreshToken)
+      .then(res => {
+        status = res.status;
+        return res;
+      })
+      .then(res => new Response(res, status));
+  }
```

Comment:
This whole thing is duplicated with `get` barring one line. I'd suggest we have a `get` which essentially does what this function is doing right now, and then a `getJSON` which calls that `get` but returns the response after converting it to JSON.

Reply:
Or if we don't want to touch the rest of the codebase, keep this as is but use it in the current `get`.

Reply:
+1